projects/wave-core-new/src/lib/operators/dialogs/point-in-polygon-filter/point-in-polygon-filter.component.html

```diff
@@ -0,0 +1,16 @@
+<wave-sidenav-header>Point in Polygon Filter</wave-sidenav-header>
+<form [formGroup]="form" (ngSubmit)="$event.preventDefault();add($event)" fxLayout="column">
```

Comment:
You could formulate a quick `<wave-dialog-help></wave-dialog-help>`

Reply:
0f336d9261962c586b6b2c8bb59b941f3ed204f7